tests/ctl/conftest.py

```diff
@@ -35,6 +37,31 @@
 TEST_CONFIG_PATH = "tests/ctl/test_config.toml"
 TEST_INVALID_CONFIG_PATH = "tests/ctl/test_invalid_config.toml"
 TEST_DEPRECATED_CONFIG_PATH = "tests/ctl/test_deprecated_config.toml"
+CONFIG = get_config()
+
+
+orig_requests_get = requests.get
+orig_requests_post = requests.post
+orig_requests_put = requests.put
+orig_requests_patch = requests.patch
+orig_requests_delete = requests.delete
+
+
+@pytest.fixture(scope="session")
+def monkeysession():
+    """monkeypatch fixture at the session level instead of the function level"""
+    mpatch = MonkeyPatch()
+    yield mpatch
+    mpatch.undo()
+
+
+@pytest.fixture(autouse=True, scope="session")
+def monkeypatch_requests(test_client, monkeysession) -> None:
```

Comment:
A quick doc string here would be good for future clarify

Reply:
Great idea